mne_bids/write.py

```diff
@@ -2543,18 +2551,24 @@ def mark_channels(bids_path, *, ch_names, status, descriptions=None, verbose=Non
             f"({len(ch_names)})."
         )
 
-    if not all(status in ["good", "bad"] for status in status):
+    if not all(status_ in ["good", "bad"] for status_ in status):
```

Comment:
[nitpick] as part of my crusade to increase usage of `set()` where appropriate, I would write this as:

```suggestion
    if set(status) != {"good", "bad"}:
```

feel free to disregard though! performance difference is negligible (41 vs 50 μs when `status` has 300 elements).

Reply:
I think this is great, thanks.